### DISCUSSION TITLE
Database Connection IP

### DISCUSSION THREAD

**Discussion by hakanatak** [open-metadata/OpenMetadata#10597] [Q&A] [answered]
I installed Openmetadata with 0.13.2 latest release on the company network and opened the relevant ports. 
Database access has the following information. "openmetadata will connect to your resource from the ip **84.51.27.125**. make sure to allow inbound traffic in your network security settings." This IP is not an IP in the network and I could not find out how it was generated. I have to use the fixed IP of the server to access the database. I have to get IP-based authorization for database access.

**Comment by dkalisch** [comment]
This is your public IP address. I have the same problem, I need to change it in a way so that openmetadata uses the internal IP....

**Comment by pmbrull** [comment, marked as answer]
Hi folks, indeed this IP is automatically fetched as the public IP.

We have been getting different feedback where this is indeed a bit confusing for some architectures, so we will disable this by default in 1.0.

In any case, you can override the value you'd like to show in this configuration https://github.com/open-metadata/OpenMetadata/blob/main/conf/openmetadata.yaml#L248.

Thanks
